src/backend/commands/extension.c

```diff
@@ -863,6 +864,7 @@ execute_extension_script(Node *stmt,
 		(void) set_config_option("log_min_messages", "warning",
 								 PGC_SUSET, PGC_S_SESSION,
 								 GUC_ACTION_SAVE, true, 0);
+	gp_enable_gpperfmon = false;
```

Comment:
Should be inside PG_TRY block as I mentioned in the #591

Reply:
Done